genie-ui/build.gradle

```diff
@@ -1,6 +1,12 @@
-plugins {
-    id "com.moowork.node" version "1.2.0"
+buildscript {
+    repositories {
+        gradlePluginPortal()
```

Comment:
Isn't this already added globally in the main `build.gradle`?

Reply:
@sghill is the one in the main `build.gradle` sufficient for these sub projects as well? You added it in both in your PR into my fork so I didn't touch it deferring to you as the expert but @mprimi might have a good point here

Reply:
Good point @mprimi @tgianos. It should delegate to the repositories defined in the parent's build.gradle.